docs/i18n/en/LC_MESSAGES/contributing.po

```diff
@@ -0,0 +1,76 @@
+# SOME DESCRIPTIVE TITLE.
+# Copyright (C) 2020-2021, The Manim Community Dev Team
+# This file is distributed under the same license as the Manim package.
+# FIRST AUTHOR <EMAIL@ADDRESS>, 2021.
```

Comment:
Is this automatically generated? Also, please follow the specific copyright and credit information as the license.

Reply:
Yes, sorry, it is automaticaly generated in each file. This is the only file that have been regenerated, so that's why the copyright is only here.

Should i remove this copyright header from every internationalized file or is it needed (with correct data of course) ?

Reply:
Nvm, it is in every files, but still: Should i remove it or leave it (with correct data) ?

Reply:
Hmm, I think it's not necessary, unless it's standard to have it.